app-ios/Modules/Sources/KMPContainer/SideEventsDataProvider.swift

```diff
@@ -0,0 +1,20 @@
+import Dependencies
+import shared
+
+public struct SideEventsDataProvider {
+    public let sideEvents: () async throws -> [SideEvent]
+}
+
+extension SideEventsDataProvider: DependencyKey {
+    public static var liveValue: SideEventsDataProvider = SideEventsDataProvider(
+        sideEvents: {
+            return SideEventKt.SideEvents
```

Comment:
(I don't know Kotlin, but) I couldn't get the SideEvent array like `SideEvent.companion.fakes()`, so I'll try to fix it!